packages/core/src/components/StepList/_StepList.docs.scss

```diff
@@ -76,6 +76,25 @@ Step object
         <p>Additional text to display under (only rendered for top-level steps)</p>
       </td>
     </tr>
+    <tr>
```

Comment:
It would be really nice to not have to manually update `StepList` documentation. In the future we should consider reorganizing `StepList` to be more like `VerticalNav`, which is able to neatly expose `VerticalNav` item object props because of the way `VerticalNavItem` is built. This will be a breaking change, but it would be nice to not only fix the documentation, but add some consistency to how we build these more complex, "nested" components. 

https://jira.cms.gov/browse/WNMGDS-386